lib/improver/tests/utilities/test_load.py

```diff
@@ -249,10 +248,7 @@ def setUp(self):
 
     def tearDown(self):
         """Remove temporary directories created for testing."""
-        Call(['rm', '-f', self.filepath])
-        Call(['rm', '-f', self.low_cloud_filepath])
-        Call(['rm', '-f', self.med_cloud_filepath])
-        Call(['rmdir', self.directory])
+        shutil.rmtree(self.directory, ignore_errors=True)
```

Comment:
Not sure about this change. shutil.rmtree(self.directory, ignore_errors=True) will try and delete everything in this directory and below. What if self.directory has been set to something stupid. The previous code only tried to remove the files it think it has created and the directory if it is empty.

Reply:
I've changed this and other similar code to use `os.remove` and `os.rmtree` functions to only remove specified files/directories.
Some of these file removals need to be wrapped in `try`/`except FileNotFoundError`/`pass` as the teardown function is shared across a group of tests where some tests create the specified files and others do not.

Reply:
@carolinejjones - if you're happy with this change, can you hit merge please?

Reply:
I'm happy with these changes. I've checked with Aaron and these changes have been tested. I shall hit the merge button